### PR TITLE
Invoices copy imrovement

### PR DESCRIPTION
**Pull request by Kovbo**
Change list:

- Add a “Pay Now” button at the bottom of each invoice in the “pending” state.
    -  When a user clicks this button, or clicks the “Pay” button from the invoices list, we should open the modal and let them enter their card details immediately. Currently there’s an error that pops up if you don’t have a payment method on file.
-  Rename the “Pending” state to “Unpaid”
-  Email copy changes
    -  Title should be “OpenPipe Invoice Dec 2023 - Jan 2024”, not “OpenPipe Usage Dec 2023 - Jan 2024”
    -  If the invoice is just for one month (ie they created their account in January) the title should end in “OpenPipe Invoice Jan 2024” instead of “OpenPipe Invoice Jan 2024 - Jan 2024”
    -  If the invoice requires payment, the button at the bottom should be “Pay Now”, not “Details”
    -  “Billing Total” should always include dollars and cents, ie. “$0.00”, not just “$0”
    

I created a script that renames the billing period if a project was created in January, and currently have the billing period as "Jan 2024 - Jan 2024":
`pnpm tsx src/server/scripts/renameBillingPeriods.ts`

### CODE REVIEW

File: app/src/components/billing/BillingContentTabs/PaymentMethods/AddPaymentMethodModal.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      return setupIntent;
+    }
+  }
+
+  return (
+    <Box style={{ width: "100%" }}>
+      <PaymentElement />
+      <HStack w="full" justifyContent="end">
+        <Button colorScheme="gray" minW={24} onClick={() => handleAddPaymentDetail()}>
+          {addPaymentDetailLoading ? <Spinner /> : "Save"}
```
Comment (corbt) [inline app/src/components/billing/BillingContentTabs/PaymentMethods/AddPaymentMethodModal.tsx]: Is this the button that I mentioned should be "Save and Pay" if you're paying right after?

Comment (Kovbo) [reply app/src/components/billing/BillingContentTabs/PaymentMethods/AddPaymentMethodModal.tsx]: Done!
I changed a toast keeping the button to make it simpler. It makes sense to change the button as well.

File: app/src/components/billing/BillingContentTabs/PaymentMethods/PayButton.tsx [hunk 2]
```diff
@@ -70,6 +70,7 @@ export default function PayButton({ invoiceId }: { invoiceId: string }) {
         clientSecret={clientSecret}
         disclosure={disclosure}
         successCallback={pay}
+        buttonText={"Pay and Save"}
```
Comment (corbt) [inline app/src/components/billing/BillingContentTabs/PaymentMethods/PayButton.tsx:73]: Save and Pay

**Comment by Kovbo** [comment]
https://github.com/OpenPipe/OpenPipe/pull/610

**Pull request closed without merging**